Makefile

```diff
@@ -267,7 +267,10 @@ rpm:
 
 	# Copy spec file to rpmbuild specs directory
 	mkdir -p $(RPMBUILDDIR)/SPECS
-	sed -e 's/@version@/$(PACKAGE_VERSION)/g; s/@release@/$(PACKAGE_RELEASE)/g' contrib/centos/aziot-identity-service.spec >$(RPMBUILDDIR)/SPECS/aziot-identity-service.spec
+	ENGINESDIR='/usr/lib64/openssl/engines'; \
+	ENGINESDIR=$$(if [ ! -d $$ENGINESDIR ]; then echo $$(openssl version -e | sed 's/^ENGINESDIR: "\(.*\)"$$/\1/'); else echo $$ENGINESDIR; fi); \
+	ENGINESDIR="$$(echo $$ENGINESDIR | sed -e 's/\//\\\//g')"; \
```

Comment:
Instead of doing this, change the sed pattern to use `|` instead of `/`.

Reply:
Nice.  Didn't know it was possible to switch the delimiter so easily.